### ISSUE TITLE
Test book build file

### ISSUE THREAD

**Issue by quaid** [theopensourceway/guidebook#44]
I just committed a very basic .adoc file with includes to have a simple way to test the build locally. This is a scratch file that can stick around the repo as long as it is useful: 5162d3d

Idea here is to have an easy way to test out some organizations in building without having to mess with a formal book file.

**Comment by semioticrobotic** [comment]
Nice, @quaid. How do you feel about the result? I see the file "compiled" as intended, but does it function/appear as you expected?

**Comment by quaid** [comment]
Yes, it's functioning as expected. The next level is to add the includes in the section files for the current version of which chapters that section will contain, then see if it builds out. When it does (should just fine right now), that gives us a baseline build target we can potentially base a regression test again. (Or at least a visual-regression test ...)

**Comment by semioticrobotic** [comment]
> Yes, it's functioning as expected. The next level is to add the includes in the section files for the current version of which chapters that section will contain, then see if it builds out. When it does (should just fine right now), that gives us a baseline build target we can potentially base a regression test again. (Or at least a visual-regression test ...)

Amazing. That kind of thing is like magic to me.
